docs/admin/auth/methods.rst

```diff
@@ -132,10 +132,17 @@ Token must contain the following claims:
 
 ``username`` - user name in a third party app.
 
-``iss``, ``username`` and ``aud`` values must match the values created by
-``CREATE USER`` statement. If ``aud`` has not been defined on the
-``CREATE USER`` statement, the cluster id is used and must match the token's
-``aud`` value. See :ref:`create-user-jwt` for details.
+The token's ``iss``, ``username`` and ``aud`` values must match the
+corresponding values within CrateDB.
+
+CrateDB uses values from :ref:`jwt_defaults` if default ``iss`` is set.
+If default ``iss`` is not defined, it will use the values specified at the
+individual user level via the ``CREATE USER`` statement.
```

Comment:
Thanks for the rephrase. But I struggle a bit with `default`, I would understand it better when calling it `global`. If this is only me, ignore
```suggestion

CrateDB uses values from :ref:`jwt_defaults` if the global ``iss`` setting is set.
If the global ``iss`` setting is not defined, it will use the values specified at the
individual user level via the ``CREATE USER`` statement.
```